custom/enikshay/integrations/utils.py

```diff
@@ -48,6 +48,18 @@ def is_valid_episode_submission(episode_case):
     return not _is_submission_from_test_location(person_case.case_id, person_case.owner_id)
 
 
+def is_migrated_uatbc_episode(episode_case):
+    try:
+        person_case = get_person_case_from_episode(episode_case.domain, episode_case)
+    except ENikshayCaseNotFound:
+        return False
+    return is_migrated_uatbc_person(person_case)
```

Comment:
@nickpell can you confirm this is the best way to figure out whether an episode originated in UATBC or eNikshay?

Reply:
```'migration_created_from_record'``` is set on episode as well

Reply:
oh, fancy!  That's handy.